### PR TITLE
add nv_codec_headers

### PR DESCRIPTION
**Pull request by ianshmean**
Related to #1384

### CODE REVIEW

**Comment by ianshmean** [comment]
Should be ready to merge

File: N/nv_codec_headers/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "nv_codec_headers"
+version = v"10.0.26-0"
```
Comment (giordano) [inline N/nv_codec_headers/build_tarballs.jl:6]: Please remove the prerelease part ("-0"), packages here should only use major, minor, and patch numbers

Comment (ianshmean) [reply N/nv_codec_headers/build_tarballs.jl]: Done. Sorry, thought that was how 4-part version numbers were handled 👍

Comment (giordano) [reply N/nv_codec_headers/build_tarballs.jl]: No, we just ignore the fourth part

**Pull request merged**